HelpSource/Guides/WritingClasses.schelp

```diff
@@ -1,10 +1,14 @@
 title:: Writing Classes
 summary:: Writing SuperCollider Classes
 categories:: Language>OOP
-related:: Reference/Classes, Classes/Class
+related:: Reference/Classes, Classes/Class, Classes/Object
 
-For a basic tutorial on how standard object-orientated classes are composed, look elsewhere
-link::http://www.google.com/search?q=oop+class+tutorial::
+
+SuperCollider follows a pure object-oriented paradigm. It is not built on data types, but on objects, which respond to messages. A class is an object that holds information about how its objects (instances) respond to such messages. Writing a class gives a definition of this behavior.
+
+What follows is not a tutorial for how to write a class system. It gives an overview of some typical expressions. See also: link::Guides/Intro-to-Objects::, link::Reference/Messages::, and link::Reference/Classes::.
```

Comment:
Maybe clearer if you write what it is before it isn't: "This is an overview of idioms used in writing classes. It is not a tutorial on writing a class system." (do you mean class hierarchy? that might be a clearer term; IME "class system" means something akin to "type system")

Reply:
ah yes.  I'm writing: "It is not a tutorial on writing a system of interrelated classes"

 (the system of related classes doesn't have to be in a hierarchy with each other).